src/services/helpers.js

```diff
@@ -9,3 +9,12 @@
 export function isNumeric(value) {
   return !isNaN(parseInt(value)) && isFinite(value)
 }
+
+export function bankersRound(x) {
+  var r = Math.round(x)
+  return (x > 0 ? x : -x) % 1 === 0.5 ? (even(r) ? r : r - 1) : r
+}
+
+function even(r) {
```

Comment:
Added some property-based tests to this, too. I'm always suspect of `%` since it's really remainder not modulo and I always worry about the ordering. 😅

Reply:
@scotttrinh Did you amend your commit? I can see your changes but GitHub doesn't detect a new commit.

Reply:
That's correct. I wasn't quite what to say in the `cz` flow, so I figured these changes don't really affect the actual content of the commit. If you'd prefer a more interesting commit history, I can break down this PR into a series of more granular commits.

Reply:
No it's cool 👍